yb-voyager/cmd/exportDataFromTarget.go

```diff
@@ -63,6 +67,17 @@ func init() {
 		"Setting the flag to `false` disables the transaction ordering. This speeds up change data capture from target YugabyteDB. Disable transaction ordering only if the tables under migration do not have unique keys or the app does not modify/reuse the unique keys.")
 }
 
+func verifySSLFlags() error {
+	if source.SSLMode != "disable" && source.SSLMode != "require" && source.SSLMode != "verify-ca" && source.SSLMode != "verify-full" {
+		return fmt.Errorf("invalid SSL mode '%s' for 'export data from target'. Please restart 'export data from target' with the --target-ssl-mode flag with one of these modes: 'disable', 'require', 'verify-ca', 'verify-full'", source.SSLMode)
+
+	}
+	if (source.SSLMode == "require" || source.SSLMode == "verify-ca" || source.SSLMode == "verify-full") && source.SSLRootCert == "" {
```

Comment:
Again, you can use `lo.Contains`to make the code more readable

Reply:
Done